pyproject.toml

```diff
@@ -5,9 +5,68 @@ build-backend = "setuptools.build_meta"
 [tool.setuptools_scm]
 write_to = "waveorder/_version.py"
 
+[project]
+name = "waveorder"
+description = "Wave-optical simulations and deconvolution of optical properties"
+readme = "README.md"
+requires-python = ">=3.10"
+license = { file = "LICENSE" }
+authors = [{ name = "CZ Biohub SF", email = "compmicro@czbiohub.org" }]
+maintainers = [
+    { name = "Talon Chandler", email = "talon.chandler@czbiohub.org" },
+    { name = "Shalin Mehta", email = "shalin.mehta@czbiohub.org" },
+]
+keywords = [
+    "simulation",
+    'optics',
+    'phase',
+    'scattering',
+    'polarization',
+    'label-free',
+    'permittivity',
+    "reconstruction-algorithm",
+    'qlipp',
+    'mipolscope',
+    'permittivity-tensor-imaging',
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
```

Comment:
Now '5 - Production/Stable'?

Reply:
I'd leave as `Development Status :: 4 - Beta` for now.